Project Walking/Source code/app/src/main/java/com/teamapricot/projectwalking/MainActivity.java

```diff
@@ -2,13 +2,94 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.content.Context;
 import android.os.Bundle;
 
+import android.preference.PreferenceManager;
+
+import org.osmdroid.config.Configuration;
+import org.osmdroid.tileprovider.tilesource.TileSourceFactory;
+import org.osmdroid.views.MapView;
+
+
+
+import android.view.View;
+import android.widget.TextView;
+
+import com.teamapricot.projectwalking.handlers.CameraHandler;
+import com.teamapricot.projectwalking.photos.PhotoController;
+
+import com.teamapricot.projectwalking.Navigation.MapController;
```

Comment:
When you move the map code to the `MapController` class, some of these imports will become unnecessary. If they're "greyed out", remove them 👍🏻

Reply:
Will be handled in #40.